docs/api/networkinterfaces/CellularInterface.md

```diff
@@ -65,7 +65,7 @@ To bring up the network interface:
 
 This example establishes connection with the cellular network using Mbed OS CellularInterface.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-cellular)](https://github.com/ARMmbed/mbed-os-example-cellular/blob/mbed-os-5.14/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_NetworkInterfaces/Cellular/)](github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_NetworkInterfaces/Cellular/main.cpp)
```

Comment:
@mprse could you change the link pointing to https://github.com/ARMmbed/mbed-os-example-cellular/blob/master/main.cpp

Don't worries about https://github.com/ARMmbed/mbed-os-examples-docs_only/tree/master/APIs_NetworkInterfaces/Cellular, I'll create a PR to remove that